v5/v5_managing_backup.ino

```diff
@@ -1392,54 +1497,40 @@ void inputManager(String command, int inputFrom) {
       digitalWrite(POWER_OUTPUT_SELECTOR, HIGH);
       delay(1000);
     }
+  } else if (command.indexOf("switch") != -1) {
+    if (command.indexOf("0") != -1)
+      digitalWrite(BATTERY_PAIR_SELECTOR, HIGH);
+    else if (command.indexOf("1") != -1)
+      digitalWrite(BATTERY_PAIR_SELECTOR, LOW);
+    else
+      println("Error in switch command");
+  } else if (command.indexOf("chargeFor") != -1) {
+    batteryChargeTime = fetchNumber(getCompleteString(command, "chargeFor"));
+    println("Battery charge time updated to : " + String(batteryChargeTime) +
+            " minutes");
+  } else if (command.indexOf("updateTime")) {
+    println("Updating time");
+    sendSMS("#setTime", "+923374888420");
   } else {
     println("Executing: " + command);
     say(command);
   }
 }
 
-bool isNum(String num) {
-  if (num.toInt() > -9999 && num.toInt() < 9999)
+bool companyMsg(String mobileNumber) {
+  if (mobileNumber.indexOf("Telenor") != -1)
     return true;
-  else
-    return false;
-}
-
-bool wapdaOn() {
-  if (digitalRead(WAPDA_STATE) == HIGH)
+  else if (mobileNumber.indexOf("Jazz") == -1 ||
+           senderNumber.indexOf("JAZZ") == -1 ||
+           senderNumber.indexOf("JAZZ OFFER") == -1 ||
```

Comment:
what is this ### senderNumber here, rejecting PR as this variable will cause error while compilation **fix it**

Reply:
yes! you are right, fixing it.